pkg/recipes/configloader/environment_test.go

```diff
@@ -177,6 +177,53 @@ func TestGetConfiguration(t *testing.T) {
 			},
 			errString: ErrUnsupportedComputeKind.Error(),
 		},
+		{
+			name: "recipe config with env resource",
```

Comment:
Do we need a new test for this or would it be sufficient add recipe config to any of the exiting tests https://github.com/radius-project/radius/blob/e717d111d1380b733d3e1fdd1dd10c0adc16b504/pkg/recipes/configloader/environment_test.go#L53

Reply:
makes sense! removed it